### PR TITLE
.NetSDK Release RSBackup 2023-04-01

### PR DESCRIPTION
**Pull request by ankur741999**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by live1206** [comment]
@ankur741999 
As offline discussed, there is a breaking change in the latest SDK. And it was due to the breaking change of swagger itself.
Previous API has the property named as [softDeleteRetentionPeriod](https://github.com/Azure/azure-rest-api-specs/blob/add28efcd3a5fd422285d992fb1ec5ee5a7a40a6/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/bms.json#L8578)
The latest API has the property named as [softDeleteRetentionPeriodInDays](https://github.com/Azure/azure-rest-api-specs/blob/80c21c17b4a7aa57f637ee594f7cfd653255a7e0/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/bms.json#L8603)

We need to add the previous property back in the custom code and make it proxy of the current one.
Like I did [here](https://github.com/Azure/azure-sdk-for-net/pull/38250/files#diff-077a76ce8b2f98afd88d94b3be3c8112d28968ea273fb068dad866cf499874c7) in your another PR, add a partial class of that type and add the previous property back. 
- get: return the current property
- set: set the current property of input value
- add EB.Never attribute to hide it from the user in the IDE

And also, please update the changelog with release content and the expecting release date.

**Comment by ArthurMa1978** [comment]
@ankur741999 please revert all changes which are not for your service

**Comment by ankur741999** [comment]
Hi @ArthurMa1978 : I can see commits from other people as well in this PR. Do you have any idea.. how did they come here? Given they are too many I am thinking of floating a separate PR.. will close this one. Let me know if you have any concern.

**Comment by live1206** [comment]
There are 2 more breaking changes popped us in the latest build:
- In previous API, it was [actionRequired](https://github.com/Azure/azure-rest-api-specs/blob/add28efcd3a5fd422285d992fb1ec5ee5a7a40a6/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/bms.json#L4762C10-L4762C24), in the latest API it is [actionsRequired](https://github.com/Azure/azure-rest-api-specs/blob/80c21c17b4a7aa57f637ee594f7cfd653255a7e0/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-04-01/bms.json#L4779). Please add the previous property back to `RecoveryServicesBackupPrivateLinkServiceConnectionState` in custom code and redirect get/set to the new property, like above comment. Also add `EB.Never` to it.
- In the previous API, there was [AzureVmWorkloadSAPHanaHSR](https://github.com/Azure/azure-rest-api-specs/blob/add28efcd3a5fd422285d992fb1ec5ee5a7a40a6/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/bms.json#L10169), and it's gone in the latest API. Please add [VmWorkloadSapHanaHsr ](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Generated/Models/VmWorkloadSapHanaHsr.cs) in the custom code and Add EB.Never to it.

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0 (2023-09-08)
 
 ### Features Added
 
+- Added a new property DistributedNodesInfo in AzureWorkloadContainerExtendedInfo, AzureVmWorkloadSQLAvailabilityGroupProtectableItem, AzureVmWorkloadProtectedItem response.
```
Comment (live1206) [inline sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md:7]: Add one more item:
Upgraded API version to 2023-04-01

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+- The property lists the nodes part of the distributed item at that time, their registration state and their ARM vm Id
+
 ### Breaking Changes
 
+- Renamed softDeleteRetentionPeriod to softDeleteRetentionPeriodInDays
+
 ### Bugs Fixed
```
Comment (live1206) [inline sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/CHANGELOG.md:14]: remove the empty entries

**Review by live1206 — COMMENTED** [review]
- Please update the version in https://github.com/Azure/azure-sdk-for-net/blob/c425c1ef94489e61b15956a3ca26845e70871fbf/sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Azure.ResourceManager.RecoveryServicesBackup.csproj#L3 to `1.1.0` to align with the changelog.
- Export API since you regenerate the code.

File: sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Custom/Models/RecoveryServicesBackupPrivateLinkServiceConnectionState.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        /// <summary>
+        /// ActionRequired
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string ActionRequired
+        {
+            get => ActionsRequired;
+            set
+            {
+                ActionRequired = value;
```
Comment (live1206) [inline sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/src/Custom/Models/RecoveryServicesBackupPrivateLinkServiceConnectionState.cs:22]:
```suggestion
                ActionsRequired = value;
```

**Review by live1206 — APPROVED** [review]

**Pull request merged**